spaceship/spec/portal/portal_stubbing.rb

```diff
@@ -1,7 +1,12 @@
 class PortalStubbing
   class << self
     def adp_read_fixture_file(filename)
-      File.read(File.join('spaceship', 'spec', 'portal', 'fixtures', filename))
```

Comment:
I am not that fond of the hardcoded file name in the `adp_read_fixture_file` implementation. Alternatives are maybe to have
* a different function for bin reading ( `adp_binread_fixture_file`)
* an explicit parameter (`adp_read_fixture_file(file, binary: false)`)

Reply:
Yes, that is one ugly piece of code.

I like the idea of two different functions (param is not obvious enough imho) and will implement this.

Reply:
This is now fixed.